plugins/nexus-repository-manager/README.md

```diff
@@ -60,7 +60,7 @@ The Nexus Repository Manager plugin displays the information about your build ar
        {/* ... */}
        {/* highlight-add-next-line */}
        <EntityLayout.Route
-         if={isNexusRepositoryAvailable}
+         if={isNexusRepositoryManagerAvailable}
```

Comment:
+1, can you update it above as well  `isNexusRepositoryAvailable` is used in line num 47 
https://github.com/janus-idp/backstage-plugins/pull/669/files#diff-89c422604a8c8efaee8c42d3fa324b2135edd767eb2f370544b6229ea44944ebL47